containers/keycloak-bootstrap/keycloak_bootstrap.py

```diff
@@ -11,6 +11,10 @@
     "admin/realms/{realm-name}/authentication/flows/{flow-alias}/executions"
 )
 
+URL_ADMIN_EXECUTIONS_EXECUTION = (
+    "admin/realms//{realm-name}/authentication/flows/{flow-alias}/executions/execution"
```

Comment:
Is this an extra slash?

```suggestion
    "admin/realms/{realm-name}/authentication/flows/{flow-alias}/executions/execution"
```

Reply:
yes, you are right - thank you!